environment.yml

```
@@ -5,3 +5,5 @@ dependencies:
 - conda-forge::jupyterlab	=	3.4.3
 - conda-forge::python	=	3.9.13
 - conda-forge::urllib3	=	1.26.9
+- conda-forge::seaborn = 0.11.2
+- conda-forge::plotnine = 0.8.0
```

```suggestion
- conda-forge::plotnine = 0.8.0

```

Remember to add new line. I wonder if `vscode` doesn't do this automatically 🤔

---

i did forget to add another line! vscode evidently does not add this new line and so ill remember this in the future!